### DISCUSSION TITLE
Lodash Babel Plugin

### DISCUSSION THREAD

**Discussion by ekoeryanto** [wix/react-native-ui-lib#1806] [Q&A] [answered]
Hey, why we don't use babel plugin for minimize lodash bundle size?

**Comment by ekoeryanto** [comment]
Ah, there are many lodash chains there

**Comment by ethanshar** [comment]
You are definitely right! 
We'll integrate the plugin in our lib
Stay tuned!

**Comment by ethanshar** [comment, marked as answer]
Issue was resolved in our latest version (6.11.0)

**Comment by solominh** [comment]
When using expo-atlas, lodash size is > 900KB. And the culprit is react-native-ui-lib. 
react-native-ui-lib code: import _ from 'lodash'
React native metro bundler still bundle all lodash
Please correct me if i am wrong.
